Libplanet/Net/Swarm.cs

```diff
@@ -2217,6 +2218,25 @@ private void ValidateSender(Peer peer)
             }
         }
 
+        private async Task CheckReplacementCacheAsync(
+            TimeSpan period,
+            CancellationToken cancellationToken)
+        {
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                try
+                {
+                    await Task.Delay(period, cancellationToken);
+                    await Protocol.CheckReplacementCacheAsync(cancellationToken);
```

Comment:
How about running this after removing some peers in `Protocol.RefreshTableAsync` instead of running it regularly?